ios/RNTPTDocumentView.h

```diff
@@ -430,6 +430,8 @@ static const PTAnnotationToolbarKey PTAnnotationToolbarKeyItems = @"items";
 
 @property (nonatomic, assign) BOOL saveStateEnabled;
 
+@property (nonatomic, copy, nullable) NSArray<NSString *> *excludedAnnotationListTypes;
```

Comment:
```suggestion
@property (nonatomic, strong, nullable) NSArray<NSString *> *excludedAnnotationListTypes;
```
All of the `NSArray<NSString *>` should have a storage type of `strong` not `copy`.